packages/browser-extension/src/scripts/dialog.js

```diff
@@ -13,6 +14,7 @@ const reasons = [
   'Page is laggy',
   'Page is not responding',
   'Popup showed up',
+  'Scroll Issues',
```

Comment:
Please, move this under `Cannot click` and rename it to `Cannot scroll`. Be sure to update all the indexes in consequence to this change.

Reply:
done